public/credits_for_images.txt

```diff
@@ -0,0 +1 @@
+favicon.ico = https://www.flaticon.com/free-icon/giftbox_1139982
```

Comment:
Tein tosiaan tämmöisen tiedoston, johon lisään kuvan, jos otan semmoisen käyttöön (olettaen, että sitä saa käyttää luvalla, jos antaa creditit esim. linkin tmv)

Ja public-kansiossa oleminen ei pitäisi haitata, koska tämä on julkista tietoa

Reply:
Jep juuri näin!